pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -26,6 +26,7 @@
     "kubernetes_full_version": "",
     "manifest_output": "manifest.json",
     "python_path": "",
+    "remote_folder": "",
```

Comment:
I think here we want to default to Hashicrops default https://developer.hashicorp.com/packer/docs/provisioners/shell#remote_folder
```suggestion
    "remote_folder": "/tmp",
```

Reply:
fixed